pkg/azureclients/vmssclient/interface.go

```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	// APIVersion is the API version for VMSS.
-	APIVersion = "2020-12-01"
+	APIVersion = "2022-03-01"
```

Comment:
do we need to update the apiversion in package name as well, e.g. L23 `"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2021-07-01/compute"` is still using `2021-07-01`.

Reply:
I have updated to latest `2021-12-01` package, actually there is no api changes between these two packages for disk functionality, that's why `2021-07-01` also works. Anyway, I have updated to latest package.